src/config/ConfigMetadata.ts

```diff
@@ -205,7 +205,7 @@ export const ConfigMetadata: {[key: string]: { type: string, defaultValue?: any
     },
     [ConfigNames.THUNDRA_TRACE_INTEGRATIONS_HTTP_RESPONSE_BODY_MASK]: {
         type: 'boolean',
-        defaultValue: false,
+        defaultValue: true,
```

Comment:
Can you also update default value for the `THUNDRA_AGENT_TRACE_INTEGRATIONS_HTTP_RESPONSE_BODY_MASK` in the `README`?